grr/server/grr_response_server/gui/ui/components/client/status_chip.ng.html

```diff
@@ -0,0 +1,6 @@
+<div *ngIf="(status$ | async) === 'online'; else offlineChip">
```

Comment:
Is the wrapping `<div>` needed? When online, this Component would render `<div><mat-chip>Online</mat-chip></div>`, but when offline it would only render `<mat-chip>Offline</mat-chip>`.

Reply:
This one is a little tricky, because if I try and remove the div, to integrate `ngIf` directive into the `<mat-chip>`, for some reason it won't work. Trying to enclose the offline chip in a `<div>` also doesn't work. Only other way that will work is to have a `<mat-chip-list>` enclosing the whole thing. What do you think?

Reply:
Please also wrap this mat-chip in a div to produce the same element hierarchy in the if and else branch. Because div is display: block per default, the display could change when online status changes.